src/features/chat/components/leftpanel/StorageContent.tsx

```
@@ -22,29 +24,39 @@ interface StorageContentProps {
+    if (!noteDetail?.conversations) return [];
+    return noteDetail.conversations.flatMap((conv) =>
+      conv.assistantMessage.generatedFiles.map((file) => ({
+        id: file.fileId,
+        label: file.fileName,
+        type: "ai",
+        fileUrl: file.downloadUrl,
+        mimeType: file.fileType,
+        ocrStatus: "completed", // 생성된 파일은 OCR 완료 상태로 간주
+      })),
+    );
+  }, [noteDetail?.conversations]);
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. The inferred dependencies did not match the manually specified dependencies, which could cause the value to change more or less frequently than expected. The inferred dependency was `noteDetail`, but the source dependencies were [noteDetail?.conversations]. Inferred less specific property than source.

/home/jailuser/git/src/features/chat/components/leftpanel/StorageContent.tsx:47:46
  45 |
  46 |   // THREAD 파일 목록 (AI 생성 파일)
> 47 |   const threadFiles = useMemo<StorageFile[]>(() => {
     |                                              ^^^^^^^
> 48 |     if (!noteDetail?.conversations) return [];
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 49 |     return noteDetail.conversations.flatMap((conv) =>
     …
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 58 |     );
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 59 |   }, [noteDetail?.conversations]);
     | ^^^^ Could not preserve existing manual memoization
  60 |
  61 |   const [selectedIds, setSelectedIds] = useState<number[]>([]);
  62 |   const [isSelectMode, setIsSelectMode] = useState(false);

(react-hooks/preserve-manual-memoization)